command/console.go

```diff
@@ -60,11 +60,7 @@ func (c *ConsoleCommand) RunContext(ctx context.Context, cla *ConsoleArgs) int {
 		return ret
 	}
 
-	diags := packerStarter.Initialize()
-	ret = writeDiags(c.Ui, nil, diags)
-	if ret != 0 {
-		return ret
-	}
+	_ = packerStarter.Initialize()
```

Comment:
This is the only actual change of this PR, right?

Reply:
yup !